src/main/java/com/autotune/common/target/kubernetes/service/impl/KubernetesServicesImpl.java

```diff
@@ -511,6 +511,11 @@ public boolean isDeploymentReady(String namespace, String deploymentName) {
         return deploymentReady;
     }
 
+    @Override
+    public boolean isPodsReady(String namespace, int podsReadyInMinutes) {
+        return false;
+    }
```

Comment:
If this is a dummy implementation to be replaced with the proper check later, can we please add a `TODO` in the comments indicating that